doc/apps/sf.rst

```diff
@@ -62,13 +65,13 @@ The program should produce the following output:
 
     Results
     -------
-    [0.78741742869032849, 0.4687548968197896, 0.24995884856075234, 0.26017773802799282]
-    [-0.38611559831595721, 0.87013137205881896, -0.8637591428902387, 0.87987666305358891]
-    [-0.90073440739802579, 0.6541730982429812, -0.17966121619754322, 0.7105191725142429]
-    [2.2181871945635283, 0.90784717914877755, -0.14608293659572136, 1.3234082450308977]
-    [0.26187400468879646, -0.24350554600924179, 0.15135585268536278, -1.3698092903244679]
-    [0.62928125081235919, -2.1680158143971759, -0.99450040689765284, 1.8511289378967497]
-    [1.9602717053077034, -1.8998580204278686, 0.23193528358577153, 1.5632428346967497]
-    [0.34002902283984326, 0.99624604317845789, -0.32804961033646657, 0.89742913874610075]
-    [-0.58834092978787944, -1.0543538025898438, 0.18153026050309107, 0.81751881123822168]
-    [1.3795212337550613, -1.0187713250385, 0.50738610139349849, 0.25778266780662962]
+    [-0.17006682173581544, -0.09825970340668835, 0.44794947054926926, -0.052141596937362866]
```

Comment:
Different results? It doesn't look like the program changed... :thinking:

Reply:
These are measurement results, so it's different every time the program runs. I'm not sure why I updated it in the docs though :thinking: Maybe I was being overly cautious.

Reply:
lol if they change every time, then why put the output in the docs? People will think they are getting the wrong answer if they ever try to run it

Reply:
Good point lol

Reply:
Clarified with a note in the latest commit.